modules/combined/test/tests/gap_heat_transfer_mortar/finite-2d-rz/finite.i

```diff
@@ -122,6 +122,19 @@ name = 'finite'
   []
 []
 
+[UserObjects]
+  [weighted_gap_uo]
+    type = LMWeightedGapUserObject
+    primary_boundary = plank_right
+    secondary_boundary = block_left
+    primary_subdomain = frictionless_primary_subdomain
+    secondary_subdomain = frictionless_secondary_subdomain
+    lm_variable = frictionless_normal_lm
+    disp_x = disp_x
+    disp_y = disp_y
```

Comment:
@gambka mentioned that the latest convention in MOOSE and apps is to use `displacements` instead of the individual components --which I think we've been inheriting from original mortar objects. We can make that change in follow-up PR when we find some time.